EIPS/eip-draft_Pull_Oracle_Interface.md

```diff
@@ -0,0 +1,122 @@
+---
+eip: xxxx
+title: Pull Oracle Interface
+author: Alliance of Decentralized Oracles (l.loya.b@gmail.com)
+discussions-to: https://github.com/ethereum/EIPs/issues/
```

Comment:
This has to be a link to an issue.

Reply:
Done! https://github.com/ethereum/EIPs/issues/2379